### ISSUE TITLE
Yr won't run unless you update Google Play services

### ISSUE THREAD

**Issue by jeekajoo** [microg/android_packages_apps_GmsCore#541]
Hi,

Is there anything we can do to bypass this ? https://framapic.org/ORa7UicSDSux/puetNwLCJx4f

Yr (https://play.google.com/store/apps/details?id=no.nrk.yr) quits when we click out the popup.

Thanks
J.

**Comment by Skrity** [comment]
We need another version bump probably. I'm on 0.2.4-111-gf1cdb48 and beta discord drops "Update Google Play Services" notification.
Here's logcat entry
```Google Play services out of date.  Requires 12451000 but found 12221403```

**Comment by jeekajoo** [comment]
I'm on 0.2.4-111-gf1cdb48 too. But it doesn't help.

**Comment by AreYouLoco** [comment]
Which version of microg core are you using?

Try to update to the latest build. I had this as well when I tried stable.

Dnia May 10, 2018 7:12:32 AM UTC, jeekajoo <notifications@github.com> napisał(a):
>Hi,
>
>Is there anything we can do to bypass this ?
>https://framapic.org/ORa7UicSDSux/puetNwLCJx4f
>
>Yr (https://play.google.com/store/apps/details?id=no.nrk.yr) quits when
>we click out the popup.
>
>Thanks
>J.

**Comment by jeekajoo** [comment]
@AreYouLoco I'm using last version https://github.com/microg/android_packages_apps_GmsCore/issues/541#issuecomment-388030613

**Comment by hugoncosta** [comment]
You'll have to wait for the next release, I have the same warning on Telegram, for example.

**Comment by tegvlarivs** [comment]
thank you guys, now i feel reassured. i find it confusing that the problem exists only on my main phone (moto g4 plus) and not on two other phones i have (moto g 2015 & moto e lte)

**Comment by lukechadwick** [comment]
Can we just set the version number to some arbitrarily high number so we don't have this issue any more or does it have to be a valid version that Google has released?

**Comment by kenogo** [comment]
@lukechadwick This hasn't been done because of its potential to break stuff.

**Comment by epon93** [comment]
Is there an ETA for the next F-droid release?

**Comment by bocekm** [comment]
@hugoncosta, regarding Telegram, see https://github.com/microg/android_packages_apps_GmsCore/issues/527#issuecomment-382394188

**Comment by tjallingran** [comment]
Does anyone know when a new (preview) release can be expected?

**Comment by maracuja-juice** [comment]
cc @mar-v-in

**Comment by maracuja-juice** [comment]
@tjallingran 
One of the codes would be this one that is extracted from the log:

```
06-10 09:30:38.495 29588 489 W GooglePlayServicesUtil: Google Play services out of date. Requires 12451000 but found 11951417
```

**Comment by maracuja-juice** [comment]
This is fixed. Update to the newest version! (0.2.4-111)

**Comment by maracuja-juice** [comment]
What is in your log? @jeekajoo

**Comment by tjallingran** [comment]
@maracuja-juice Thanks, but as far as I understand,  the `12451000` is the result of the calculation (and even then, the last part of that number may differ, I think).

The calculation has (at best) two unknowns, and there's not enough info (that I know about) to determine both of them.

**Comment by maracuja-juice** [comment]
I see @tjallingran
How about doing it like in this commit? 
https://github.com/NoGooLag/android_packages_apps_GmsCore/blob/1dbdcaf4ccf2806aac44dbea1916b9eb6821fc6a/play-services-core/build.gradle

**Comment by jeekajoo** [comment]
fixed in v0.2.5.12879
thanks @mar-v-in

**Comment by farhaj29** [comment]
Guys I have some trouble with GMs core v.2.5 that I can't properly update it with its other components.
